kubernetes/argocd/stacks/sparkhacks/sparkhacks-2023.yml

```diff
@@ -63,6 +63,7 @@ metadata:
 spec:
   entryPoints:
     - websecure
+    - web
```

Comment:
If you are doing a redirect, then you wouldn't add an entrypoint here. You'd likely use another `IngressRoute` with some redirect logic instead.

Reply:
If we add redirects for the entire `web` entry point, this probably doesn't need to be here unless you still expect to serve the actual website over HTTP.